easy_rec/python/main.py

```diff
@@ -247,20 +247,24 @@ def _train_and_evaluate_impl(pipeline_config, continue_train=False):
     if 'cluster' in tf_config and 'ps' in tf_config['cluster'] and (
```

Comment:
add a environment variable to differentiate whether it is Data Science environment.